build/specifications/Docker.json

```diff
@@ -8270,6 +8271,238 @@
+            "type": "long?",
+            "format": "--cpu-shares {value}",
+            "help": "CPU shares (relative weight)."
+          },
+          {
+            "name": "CpusetCpus",
+            "type": "string",
+            "format": "--cpuset-cpus {value}",
+            "help": "CPUs in which to allow execution (0-3, 0,1)."
+          },
+          {
+            "name": "CpusetMems",
+            "type": "string",
+            "format": "--cpuset-mems {value}",
+            "help": "MEMs in which to allow execution (0-3, 0,1)."
+          },
+          {
+            "name": "File",
+            "type": "string",
+            "format": "--file {value}",
+            "help": "Name of the Dockerfile (Default is <c>PATH/Dockerfile</c>)."
+          },
+          {
+            "name": "Iidfile",
+            "type": "string",
+            "format": "--iidfile {value}",
+            "help": "Write the image ID to the file."
+          },
+          {
+            "name": "Isolation",
+            "type": "string",
+            "format": "--isolation {value}",
+            "help": "Container isolation technology."
+          },
+          {
+            "name": "Label",
+            "type": "List<string>",
+            "format": "--label {value}",
+            "help": "Set metadata for an image."
+          },
+          {
+            "name": "Load",
+            "type": "bool",
+            "format": "--load",
+            "help": "Shorthand for <c>--output=type=docker</c>"
+          },
+          {
+            "name": "Memory",
+            "type": "long?",
+            "format": "--memory {value}",
+            "help": "Memory limit."
+          },
+          {
+            "name": "MemorySwap",
+            "type": "long?",
+            "format": "--memory-swap {value}",
+            "help": "Swap limit equal to memory plus swap: <c>-1</c> to enable unlimited swap."
+          },
+          {
+            "name": "Network",
+            "type": "string",
+            "format": "--network {value}",
+            "help": "Set the networking mode for the RUN instructions during build."
+          },
+          {
+            "name": "NoCache",
+            "type": "bool",
+            "format": "--no-cache",
+            "help": "Do not use cache when building the image."
+          },
+          {
+            "name": "Output",
+            "type": "string",
+            "format": "--output {value}",
+            "help": "Output destination (format: <c>type=local,dest=path</c>)."
+          },
+          {
+            "name": "Platform",
+            "type": "string",
+            "format": "--platform {value}",
+            "help": "Set platform if server is multi-platform capable."
+          },
+          {
+            "name": "Progress",
+            "type": "string",
+            "format": "--progress {value}",
+            "help": "Set type of progress output (auto, plain, tty). Use plain to show container output."
```

Comment:
But I think here we can use an enumeration :)

Reply:
Also added enumeration for older build command